lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -400,8 +261,6 @@ module Resources
               'display': 'child',
               'userSelected': false
             }
-          ],
-          'text': 'Child'
```

Comment:
Sorry, this needs to be added back as well.

Reply:
added back https://github.com/cerner/fhir.cerner.com/pull/624/commits/0105b4346c586ef01f5a2008435fd01408993d76#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR264-R265